index.html

```diff
@@ -17,148 +17,255 @@
             <a class="logo-container" href="">
                 <img src="images/image_geometry_1logo.png" alt="logo">
             </a>
-
             <nav>
+                <a href="#Hello">Hello</a>
+                <a href="#projects">Portfolio</a>
+                <a href="#about">About</a>
+                <a href="#contact">Contact</a>
+
```

Comment:
- [ ] [OPTIONAL] When I hover these `a` tags all the websites move, it would be nice if you fix it 😉  

![image](https://user-images.githubusercontent.com/60164239/131208410-c0b2571c-686b-4ab9-9998-7e7537d20044.png)